filclient.go

```diff
@@ -982,9 +989,8 @@ func (fc *FilClient) RetrieveContent(ctx context.Context, miner address.Address,
 				log.Debugf("unrecognized voucher response type: %v", resType)
 			}
 		case datatransfer.DataReceivedProgress:
-			if fc.RetrievalProgressLogging && time.Since(lastReceivedUpdate) >= time.Millisecond*100 {
-				fmt.Printf("received: %v (%v)\r", state.Received(), humanize.IBytes(state.Received()))
-				lastReceivedUpdate = time.Now()
+			if progressCallback != nil {
```

Comment:
slightly nicer pattern is to check if progressCallback is nil at the top of the function, and if it is, set it equal to `func(..) {}`